relay-server/src/metrics_extraction/spans/mod.rs

```diff
@@ -964,20 +964,6 @@ mod tests {
                         "db.operation": "SELECT"
                     }
                 },
-                {
-                    "description": "SELECT 'TABLE'.'col' FROM 'TABLE' WHERE 'TABLE'.'col' = %s",
```

Comment:
Removed this example because it is AFAIK uncommon to wrap identifiers in single quotes in SQL.